docs/error_pages/404.rst

```diff
@@ -0,0 +1,8 @@
+ Error 404
+===========
```

Comment:
It looks like they shouldn't be there, as this is rendering as a header (and there is no title in the generate document).